### PR TITLE
Adds Stage and Waterpool structures

### PR DESCRIPTION
**Pull request by tech45**
Please open readme.md and click on view button to see the Task
This is for my GCI Task of terasology
Please wait when you have clicked the view button in readme.md as it takes time to load Images . I have included images in readme.md

### CODE REVIEW

**Review by pollend — CHANGES_REQUESTED** [review]
can you remove the extra `readme.md` and move the previews to the main `README.md`.

**Comment by tech45** [comment]
I have removed the readme.md

**Comment by tech45** [comment]
now I will move my previews to the main README.md
Now please refer to changes below this comment.

**Comment by tech45** [comment]
Sir I have revised my work and done changes according to your suggestion. Hope you will like it.
Refer to changes above this comment

File: README.md [hunk 1]
```diff
@@ -10,3 +10,39 @@ Also add a picture of the structure to this [README](README.md) file.
 
 ### Well Entrance
 ![Well Entrance](https://github.com/Terasology/Mineshafts/raw/master/preview/WellEntrance.jpg)
+
+# This is for my GCI Task (I) - Structure Templates.
```
Comment (pollend) [inline README.md]: can you remove all this extra information about your gci task. format your images similarly to how it's formatted in the main repository. If you really want the description you can add it below the image.

**Review by pollend — CHANGES_REQUESTED** [review]

**Comment by tech45** [comment]
Sir I have Done changes , Please refer it above this comment.

File: README.md (hunk 1 above)
Comment (tech45) [reply README.md]: Sir see below , I have done changes and updated it,

File: README.md [hunk 2]
```diff
@@ -10,3 +10,38 @@ Also add a picture of the structure to this [README](README.md) file.
...
+### Waterpool View - 2
+![Waterpool view-2](https://github.com/tech45/AdditionalStructureTemplates/blob/master/preview/Waterpool%20view-2.png)
+
+### More Waterpool using Structure Spawner
+![More Waterpool](https://github.com/tech45/AdditionalStructureTemplates/blob/master/preview/MoreWaterpools.png)
+
+
+# This is for my GCI Task (I) - Structure Templates.
+
+I have created Two new and awsome Structures!
```
Comment (pollend) [inline README.md]: The readme is what people are going to see when they visit the repository. not sure why you need to refer to yourself in this way. can you remove this bottom section.

**Review by pollend — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tech45) [reply README.md]: Sir you can see that  I have done changes

**Comment by tech45** [comment]
Sir please refer to the final changes just above this comment.

**Comment by pollend** [comment]
open this back up please.

**Comment by tech45** [comment]
Sir I have opened this PR again as per your request

**Pull request merged**